src/coffee/io/reader.coffee

```diff
@@ -6,18 +6,26 @@ iconv = require 'iconv-lite'
 fs = Promise.promisifyAll require('fs')
 Excel = require 'exceljs'
 
+debugLog = _.noop
+
 class Reader
 
   constructor: (@options = {}) ->
```

Comment:
should be able to pass option from CLI to turn on debugging - for example: `--debug`